### ISSUE TITLE
OS X Docker Quickstart Timeout

### ISSUE THREAD

**Issue by crockpotveggies** [docker/toolbox#495]
_OS X El Capitan all updates installed_

Downloaded Docker toolbox installer and upgraded. Docker quickstart terminal successfully opens in my terminal window, but the VM machine "default" is timed out, I can't run any Docker functions:

```
bash --login '/Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts/start.sh'
Justins-MacBook-Air:~ justin$ bash --login '/Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts/start.sh'
Starting "default"...
(default) Check network to re-create if needed...
(default) Waiting for an IP...
Machine "default" was started.
Waiting for SSH to be available...
Detecting the provisioner...
Started machines may have new IP addresses. You may need to re-run the `docker-machine env` command.
Regenerate TLS machine certs?  Warning: this is irreversible. (y/n): Regenerating TLS certificates
Waiting for SSH to be available...
Detecting the provisioner...
Copying certs to the local machine directory...
Copying certs to the remote machine...
Setting Docker configuration on the remote daemon...
Error checking TLS connection: Error checking and/or regenerating the certs: There was an error validating certificates for host "192.168.99.100:2376": tls: DialWithDialer timed out
You can attempt to regenerate them using 'docker-machine regenerate-certs [name]'.
Be advised that this will trigger a Docker daemon restart which will stop running containers.


                        ##         .
                  ## ## ##        ==
               ## ## ## ## ##    ===
           /"""""""""""""""""\___/ ===
      ~~~ {~~ ~~~~ ~~~ ~~~~ ~~~ ~ /  ===- ~~~
           \______ o           __/
             \    \         __/
              \____\_______/


docker is configured to use the default machine with IP 192.168.99.100
For help getting started, check out the docs at https://docs.docker.com

Justins-MacBook-Air:~ justin$ docker-machine ls
NAME      ACTIVE   DRIVER       STATE     URL   SWARM   DOCKER   ERRORS
default            virtualbox   Timeout
```

**Comment by crockpotveggies** [comment]
It would appear the Timeout is not actually a timeout, but the VM starting up the docker daemon is taking *extra* long to bootup. Can the message be changed? Also, I had to close and re-open the quickstart terminal in order to successfully access the VM and Docker daemon.

**Comment by coyoteconscious** [comment]
When first installing the tools, I was getting the timeout error. I did not have iTerm2 installed on the machine yet, assuming it would be possible to use terminal for the moment. Apparently, at least on my machine running Sierra, it would fail during the process of checking for the presence of iTerm on the machine, then claim a timeout had occurred. I installed and ran iTerm2, and now it seems to work.
